se/se_epub_lint.py

```diff
@@ -591,6 +591,21 @@ def _recursive_helper(rules: cssutils.css.cssstylesheet.CSSStyleSheet, local_css
 
 	return (local_css_rules, duplicate_selectors)
 
+def files_not_in_spine(self) -> set:
+	"""
+	Check the spine against the actual files.
+
+	INPUTS
+	None
+
+	OUTPITS
+	Set of files not in the spine (typically an empty set)
+	"""
+
+	xhtml_files = set(self.path.glob("src/epub/text/*.xhtml"))
```

Comment:
Instead of hard-coding `/src/epub/text/*.xhtml` we should use `f"{self.content_path}/**/*.xhtml"` (that is, if `glob` accepts `**`, I don't know if it does. If it doesn't then we should do a recursive search for xhtml files instead of hard coding the `text` folder.)  The `text` folder is an SE convention, not an epub requirement.

Reply:
`toc.xhtml` isn’t in the spine. I can hardcode an exception there, but do you know of any other exceptions?

Reply:
You can add it separately using `self.toc_path`